README.md

```diff
@@ -24,3 +24,8 @@ by multiple sources.
 
 See our [L3AF Development Environment](https://github.com/l3af-project/l3af-arch/tree/main/dev_environment)
 for a quick and easy way to try out L3AF on your local machine.
+
```

Comment:
I will also point out that https://wiki.lfnetworking.org/display/L3AF/Governance says
"The list of voting members of the TSC will be maintained on the “CONTRIBUTING” file within the Project’s code repository." but that information is not here, nor is there any pointer to it from here.   My opinion is this isn't the right page for it, but I wanted to point that out so we can discuss.